index.html

```diff
@@ -2869,8 +2869,9 @@ <h3>Production</h3>
 
             <p>
 The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD
-specification</a> defines values that are valid for this entry. This entry
-contains representation-specific syntax and therefore could be present in the <a
+specification</a> defines values that are valid for this member. It
+contains representation-specific syntax and therefore could be present as an
+entry in the <a
 href="#data-model">data model</a> to aid in lossless conversion. If the entry
```

Comment:
I addressed some of this problematic language here: https://github.com/w3c/did-core/pull/626

I believe it's compatible with this PR.